### PR TITLE
Added overview of validated patterns and removed the old content

### PR DESCRIPTION
**Pull request by hbisht-RH-ccs**
[TELCODOCS-1987](https://issues.redhat.com//browse/TELCODOCS-1987): Revising About Validated Patterns content

Replaced the old content "About validated patterns" with the new "Overview of validated patterns" content.
It talks about the Overview, Relationship to reference architectures, problem Validated Patterns solve, goals, Who should use Validated Patterns?, community and ecosystem, Red Hat’s involvement, and application deployment workflows.

### CODE REVIEW

**Comment by mbaldessari** [comment]
This is an automated message:

You can preview this docs PR at http://469.docs-pr.validatedpatterns.io
Note that they get generated every five minutes, so please wait a bit.

**Comment by mbaldessari** [comment]
The 503 failures on redhat.com is because the website is down, so those can be ignored at the moment

File: content/learn/about-validated-patterns.adoc [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+
+The diagram below outlines the general deployment flow for a data center application. Before proceeding, users must create a fork of the pattern repository to allow for changes to operational elements (such as configurations) and application code. These changes can then be successfully pushed to the forked repository as part of DevOps continuous integration (CI). Clone the repository to your local machine, and push future changes to your fork.
+
+
+image::/images/gitops-datacenter.png[GitOps for Datacenter]
+
+. Make a copy of the values file, for example, `values-global.yaml` or `values-hub.yaml`. These values files specify subscriptions, operators, applications, and other details. Additionally, each Validated Pattern contains a `values-secret` template file, which provides secret values required to successfully install the pattern. Patterns do not require committing secret material to git repositories. It is important to *avoid pushing sensitive information to a public repository accessible to others*. The Validated Patterns framework includes components to facilitate the safe use of secrets.
```
Comment (mhjacks) [inline content/learn/about-validated-patterns.adoc:80]: Possible edit: "In your fork, if needed, edit the values files, such as `values-global.yaml` or `values-hub.yaml`, to customize or personalize your deployment."

**Review by mhjacks — CHANGES_REQUESTED** [review]
Sorry, I missed this in the pre-PR review. Any editing of the values files would be done in the fork - we wouldn't want them making copies as the framework wouldn't "see" the copies unless they made other changes.

File: content/learn/about-validated-patterns.adoc [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+[id="general-structure"]
+=== General structure
+
+All patterns assume an available OpenShift cluster for deploying applications. If no OpenShift cluster is available, you can use https://console.redhat.com/openshift[cloud.redhat.com]. The documentation uses the `oc` command syntax, but `kubectl` can be used interchangeably. For each deployment, it is assumed that the user is logged into a cluster using the `oc login` command or by exporting the `KUBECONFIG` path.
+
+
+The diagram below outlines the general deployment flow for a data center application. Before proceeding, users must create a fork of the pattern repository to allow for changes to operational elements (such as configurations) and application code. These changes can then be successfully pushed to the forked repository as part of DevOps continuous integration (CI). Clone the repository to your local machine, and push future changes to your fork.
```
Comment (kquinn1204) [inline content/learn/about-validated-patterns.adoc:75]: The following diagram ...

File: content/learn/about-validated-patterns.adoc [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+. Deploy the application as specified by the pattern, usually by using a `make` command (`make install`). When the workload is deployed, the pattern first deploys the Validated Patterns operator, which in turn installs OpenShift GitOps. OpenShift GitOps then ensures that all components of the pattern, including required operators and application code, are deployed.
+
+Most patterns also include the deployment of an Advanced Cluster Management (ACM) operator to manage multi-cluster deployments.
+
+
+[id="edge-patterns"]
+=== Edge Patterns
+
+Many patterns include both a data center and one or more edge clusters. The diagram below outlines the general deployment flow for applications on an edge cluster. Edge OpenShift clusters are typically smaller than data center clusters and might be deployed on a three-node cluster that allows workloads on master nodes, or even on a single-node cluster (SNO). These edge clusters can be deployed on bare metal, local virtual machines, or in a public or private cloud.
```
Comment (kquinn1204) [inline content/learn/about-validated-patterns.adoc:91]: The following diagram ..

**Review by kquinn1204 — COMMENTED** [review]
great work some minor comments but flow nicely

File: content/learn/about-validated-patterns.adoc (hunk 1 above)
Comment (hbisht-RH-ccs) [reply content/learn/about-validated-patterns.adoc:80]: Done, as suggested. Thanks!

**Pull request merged**